### ISSUE TITLE
onagre fails to start: SIGSEGV (Address boundary error)

### ISSUE THREAD

**Issue by dbarnett** [onagre-launcher/onagre#86]
**Describe the bug**
onagre crashes on startup for me with a SIGSEGV error and no other context, 100% reproducible.

**Config** 
No config, fresh install of onagre 1.1.0.
Also tried it with 1.0.0 and build from source at 543f1fe8, same results.

**To Reproduce**
Run `onagre`, see error.

**Expected behavior**
onagre should run successfully and show me an actual launcher, OR tell me explicitly if there's some problem with my installation or environment preventing it from starting.

**Actual behavior**
Crashes on startup:
```shell
$ onagre 
2024-09-05T19:44:52.300091Z  INFO onagre: Starting onagre
2024-09-05T19:44:52.300255Z ERROR onagre::app::style: Failed to parse theme "/home/david/.config/onagre/theme.scss": Failed to open config file
2024-09-05T19:44:52.300270Z  WARN onagre::app::style: Failing back to default theme
fish: Job 1, 'onagre' terminated by signal SIGSEGV (Address boundary error)
```

**Window manager (please complete the following information):**
 - Window manager: GNOME Shell (on Wayland)
 - OS: Ubuntu Linux 24.04

**Comment by oknozor** [comment]
Can you provide a bit more context (config file) ? 
Do you have pop-launcher installed ?

**Comment by oknozor** [comment]
Vulkan installed ?

**Comment by oknozor** [comment]
Please try to run with `WGPU_BACKEND=gl  onagre`. 
Alternatively try to install vulkan.  

Also can you provide the output of `RUST_LOG=debug onagre`

**Comment by dbarnett** [comment]
Derp, now it's working fine. No idea what changed but I couldn't repro any issues when I went to try it with those options...

Thanks for your help! If I see errors again I'll definitely follow up with such troubleshooting details, and I might try deleting everything and reinstalling just to check if I can unfix whatever got fixed... I strongly believe that software should never be crashing with unexplained SIGSEGV out of the box, no matter what way users may be holding it. =)
